iam.tf

```diff
@@ -71,14 +71,14 @@ resource "databricks_group_member" "this" {
 
 resource "databricks_entitlements" "this" {
   for_each = {
-    for group, params in var.iam : group => params.entitlements
+    for group, params in var.iam : group => params
     if params.entitlements != null
```

Comment:
try to comment out this string because this check prevents creation of map, however we have discussed that we should create it even if none 'entitlements' are provided

Reply:
removed and tested